### PR TITLE
Updated with node-12

### PR DESCRIPTION
**Pull request by kazweda**
I run "npm install" with node(v12.13.0) and I got some errors.
Then I removed package-lock.json and run "npm install" again.
After that, package.json and package-lock.json were updated.

### CODE REVIEW

**Comment by kazweda** [comment]
I'm not sure but should I append `node_js: - 12` in `.travis.yml` ?

File: package.json [hunk 1]
```diff
@@ -44,5 +44,8 @@
     "jest": "^24.0.0",
     "prettier": "^1.16.0",
     "rewire": "^4.0.1"
+  },
+  "dependencies": {
+    "acorn": "^7.1.0"
   }
```
Comment (twada) [inline package.json]: `acorn` is a parser so it shouldn't be in dependencies (should be in devDependencies).

File: .travis.yml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 language: node_js
 node_js:
+    - 12
     - 11
     - 10
     - 9
```
Comment (twada) [inline .travis.yml]: We'd better use LTS releases only, so it should be 

```yml
node_js:
    - 12
    - 10
```



https://nodejs.org/ja/about/releases/

**Comment by twada** [comment]
LGTM 👍

**Pull request merged**